app/src/components/users/UserInfoForm.vue

```diff
@@ -121,17 +132,27 @@ export default class AddUserForm extends Vue {
       })
   }
 
-  updateUser (field: string, value: string): void {
-    this.$set(this.newUserData, field, value)
+  saveUser (): void {
+    axios
```

Comment:
I suggest using ES7 syntax for handle asynchronous code and replacing .then with async/await

Reply:
the first article I found with a pretty good explanation of why async/await is better)
https://www.smashingmagazine.com/2020/11/comparison-async-await-versus-then-catch/